### PR TITLE
capistrano config

### PR DESCRIPTION
**Pull request by nicosomb**
Used for v2 deployment.

### CODE REVIEW

**Comment by j0k3r** [comment]
You don't need other specific configuration? Just the default one is ok?

**Comment by nicosomb** [comment]
I also have a config directory (with deploy.rb, etc.) ... but I forgot to add them in gitignore. Will do.

**Comment by nicosomb** [comment]
Fixed .gitignore.

**Comment by j0k3r** [comment]
And you still didn't add the config files :)

**Comment by j0k3r** [comment]
So, with this config, you just do `cap deploy`, right?

**Comment by nicosomb** [comment]
`bundle exec cap staging deploy`

**Comment by j0k3r** [comment]
This should need some documentation for the v2 (ping #1359)

File: .gitignore [hunk 1]
```diff
@@ -36,3 +36,8 @@
 # Data for wallabag
 data/assets/*
 data/db/wallabag*.sqlite
+
+# Capistrano
+/config/deploy.rb
+/config/deploy/production.rb
+/config/deploy/staging.rb
```
Comment (j0k3r) [inline .gitignore]: Is that necessary since you finally add them?

**Pull request merged**